docs/components/messageListener.md

```diff
@@ -0,0 +1,168 @@
+<!--
+Copyright (c) 2020 The Gamepad Navigator Authors
+See the AUTHORS.md file at the top-level directory of this distribution and at
+https://github.com/fluid-lab/gamepad-navigator/raw/master/AUTHORS.md.
+
+Licensed under the BSD 3-Clause License. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the BSD 3-Clause License at
+https://github.com/fluid-lab/gamepad-navigator/blob/master/LICENSE
+-->
+
+# `gamepad.messageListener`
+
+This component listens to the messages and signals sent by the [`inputMapper`](inputMapper.md) component in the content
+scripts and then calls the action invokers according to the received message.
+
+## Using this grade
+
+The component can be used by creating its instance or by making a custom component using the grade.
+
+``` javascript
+// Either create an instance of the messageListener.
+var messageListenerInstanceOne = gamepad.messageListener();
+
+// Otherwise create a custom component using the messageListener grade.
+fluid.defaults("my.messageListener.grade", {
+    gradeNames: ["gamepad.messageListener", "my.CustomGrade"]
+});
+var messageListenerInstanceTwo = my.messageListener.grade();
+```
+
+## Component Options
+
+This component does not provide any configuration options.
+
+## Non-navigation Invokers
+
+### `{messageListener}.addListener()`
```

Comment:
If you are making use of the `addListener` invoker inherited from `fluid.component`, you should mention that and provide a link to that documentation.

Reply:
It's not the `addListener` from `fluid.component`. Rather it uses the one provided by Chrome Extension APIs, i.e.  `chrome.runtime.onMessage.addListener` which listens to the signals from content scripts. But sure, I'll add a link to its documentation.